src/sentry/static/sentry/app/views/organizationHealth/devices.jsx

```diff
@@ -1,62 +1,89 @@
 import {Flex} from 'grid-emotion';
+import PropTypes from 'prop-types';
 import React from 'react';
 
 import {t} from 'app/locale';
+import OrganizationHealthDetails from 'app/views/organizationHealth/details';
 import PieChart from 'app/components/charts/pieChart';
 
+import DetailContainer from './detailContainer';
 import EventsTableChart from './eventsTableChart';
-import Header from './styles/header';
 import HealthPanelChart from './styles/healthPanelChart';
 import HealthRequest from './util/healthRequest';
-import withHealth from './util/withHealth';
+
+class OrganizationHealthDevicesOverview extends React.Component {
+  static propTypes = {
+    tag: PropTypes.string,
+    title: PropTypes.string,
+    onSetSpecifier: PropTypes.func,
+  };
+
+  render() {
+    const {title, tag, onSetSpecifier} = this.props;
+    return (
+      <HealthRequest
+        tag={tag}
+        showLoading
+        includeTimeseries={false}
+        includeTop
+        includePercentages
+        limit={5}
+      >
+        {({tagData, tagDataWithPercentages}) => (
+          <React.Fragment>
+            <Flex>
+              <HealthPanelChart
+                height={300}
+                showLegend={false}
+                series={[
+                  {
+                    seriesName: title,
+                    data: tagData.map(([name, value]) => ({name, value})),
+                  },
+                ]}
+                title={title}
+              >
+                {({series}) => <PieChart height={300} selectOnRender series={series} />}
```

Comment:
The selectOnRender behaviour is quite confusing to me (i.e. how after you move your mouse over it, it disappears permanently)

Reply:
Yeah it's something we should address in the future, unfortunately ECharts doesn't provide any great behavior by default and we'll have to develop something ourselves.